### PR TITLE
Add documentation for installing CUDA

### PR DESCRIPTION
**Pull request by lukeyeager**
Enough questions have come up that I feel like this deserves its own doc.

### CODE REVIEW

File: docs/InstallCuda.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+On Ubuntu, you can install a driver in two ways: with a `run` file or with a Debian package.
+
+It is recommended that you use a Debian package to install your driver, unless you have a new GPU that requires a newer driver version.
+Debian packages are simpler to install, uninstall and upgrade, while `run` file installers are useful if you need a newer driver version.
+
+To install with a `run` file, download one from the [NVIDIA Driver Downloads](http://www.nvidia.com/Download/index.aspx) website and follow the instructions.
+If you run into any problems, look at the "Additional Information" section.
+
+> **IMPORTANT**: If you use a `run` file to install your driver, don't install the `cuda` Debian package.
+More information [below](#deb-packages).
```
Comment (gheinrich) [inline docs/InstallCuda.md]: shouldn't it be `#debian-packages`?

**Comment by gheinrich** [comment]
Thanks for the doc!

File: docs/InstallCuda.md (hunk 1 above)
Comment (lukeyeager) [inline docs/InstallCuda.md]: Yeah we keep waffling on this throughout the documentation. Scanning through our other documentation [here](https://developer.nvidia.com/cuda-downloads) and [here](http://docs.nvidia.com/cuda/cuda-installation-guide-linux), it looks like we're not consistent there, either. Although Andy does overall seem to prefer "RPM vs. Deb" to "RPM vs. Debian".

In my mind, "deb" is an archive format. You can use some packaging tools to build "Debian deb packages" for a Debian distro, or "Ubuntu deb packages" for a Ubuntu distro.

It seems like a matter of opinion to me - I can't find anything that makes a case for "the right way" to talk about it.

Comment (gheinrich) [inline docs/InstallCuda.md]: Sure I don't have a strong opinion either - actually what I meant really is the link is broken :-)

Comment (lukeyeager) [inline docs/InstallCuda.md]: I decided to go with "Deb" like we do in the CUDA installation docs.

**Pull request merged**